google/cloud/bigtable/examples/bigtable_samples.cc

```diff
@@ -310,21 +312,62 @@ void SampleRowsCollections(google::cloud::bigtable::Table table) {
 }
 //! [sample row keys collections]
 
+//! [run table operations]
+void RunTableOperations(google::cloud::bigtable::TableAdmin admin,
+                        std::string const& table_id) {
+  // Create the table
+  std::cout << std::endl << "Creating a table: ";
+  auto schema = admin.CreateTable(
+      table_id,
+      google::cloud::bigtable::TableConfig(
+          {{"fam", google::cloud::bigtable::GcRule::MaxNumVersions(10)},
+           {"foo",
+            google::cloud::bigtable::GcRule::MaxAge(std::chrono::hours(72))}},
+          {}));
+  std::cout << " Done" << std::endl;
+
+  // List tables
```

Comment:
Note that this comment (and other similar comments) repeat the information in the status messages that follow them, I suggest removing the comments.